testing_dir/minesweeper.py

```diff
@@ -89,7 +89,7 @@ def game_loop():
             player_pos = new_pos
             moves += 1
 
-        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES:
+        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES + 1:  # Error: should not have +1
```

Comment:
The comparison condition may be incorrect for winning the game as it should not have `+1`. Ensure that this logic correctly reflects the game's win condition.
```diff
-        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES + 1:  # Error: should not have +1
+        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES:
```